### DISCUSSION TITLE
Should I add my noreply email address?

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#23206] [New to GitHub]
<p>Should I add my noreply email address to the list of email addresses associated with my GitHub account? Or is the fact that I’ve selected the <strong>Block command line pushes that expose my email</strong> and configured my local Git repo using the noreply email, sufficient?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Your noreply email is associated with your account by default, so you don’t have to do anything with your GitHub settings.</p>
<p>You do want to make sure that your git config settings on your own machine are set to use the noreply email address. You should be able to check that by running this command:</p>
<pre><code> git config user.email</code></pre>

**Comment by rfalanga** [comment]
<p>My user.email is using my GitHub Noreply email. Thank you, <a class="mention" href="/canuckjacq">@canuckjacq</a></p>

**Comment by xucian** [comment]
are commits having this no reply email (id+username@users.noreply.github.com) supposed to affect the activity graph on a user's profile?
because it seems that's not the case. I think it worked before, but today I suddenly realized my activity graph is only shows my web-based contributions..
I have my emails set as private and I double-checked my local user.email -- it's the correct one
